pkg/querier/queryrange/codec.go

```diff
@@ -805,6 +817,19 @@ func toProtoVector(v loghttp.Vector) []queryrangebase.SampleStream {
 	return res
 }
 
+func toProtoScalar(v loghttp.Scalar) []queryrangebase.SampleStream {
+	res := make([]queryrangebase.SampleStream, 0, 1)
+
+	res = append(res, queryrangebase.SampleStream{
+		Samples: []logproto.LegacySample{{
+			Value:       float64(v.Value),
+			TimestampMs: v.Timestamp.UnixNano() / 1e6,
```

Comment:
nit: There is a `v.Timestamp.UnixMilli()` function for that.

Reply:
Ah, right, this is a `prometheus/common/model.Time` so there's no UnixMilli() available